set.mm

```diff
@@ -472151,6 +472186,240 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+       e. F Z = ( x .x. y ) ) ) $=
+      ( mgpbas mgpplusg elgrpsumset ) ABCDFGHIJKDEJNLREFJNMSOPQT $.
+  $}
+
+  ${
+    ringsumsetss.1 $e |- B = ( Base ` R ) $.
+    ringsumsetss.3 $e |- G = ( mulGrp ` R ) $.
+    ringsumsetss.4 $e |- .X. = ( +f ` G ) $.
+    ringsumsetss.5 $e |- ( ph -> R e. Ring ) $.
+    ringsumsetss.6 $e |- ( ph -> E C_ B ) $.
+    ringsumsetss.7 $e |- ( ph -> F C_ B ) $.
+    $( Closure for the sumset multiplicative operation in a ring.  (Contributed
+       by Thierry Arnoux, 20-Jan-2024.) $)
+    ringsumsetss $p |- ( ph -> ( .X. " ( E X. F ) ) C_ B ) $=
+      ( mgpbas crg wcel cmnd ringmgp syl mndsumsetss ) ABDEFGBCGIHNJACOPGQPKCGI
+      RSLMT $.
+  $}
+
+  ${
+    $d .X. x y $.  $d A x $.  $d B x y $.  $d K x y $.  $d R x y $.
+    $d X x y $.  $d h ph $.  $d h x $.  $d ph x y $.
+    sumsetsnpridl.1 $e |- B = ( Base ` R ) $.
+    sumsetsnpridl.2 $e |- G = ( mulGrp ` R ) $.
+    sumsetsnpridl.3 $e |- .X. = ( +f ` G ) $.
+    sumsetsnpridl.4 $e |- K = ( RSpan ` R ) $.
+    sumsetsnpridl.5 $e |- ( ph -> R e. Ring ) $.
+    sumsetsnpridl.6 $e |- ( ph -> X e. B ) $.
+    $( The sumset of a ring with a single element is the element's principal
+       ideal.  (Contributed by Thierry Arnoux, 21-Jan-2024.) $)
+    sumsetsnpridl $p |- ( ph -> ( .X. " ( B X. { X } ) ) = ( K ` { X } ) ) $=
+      ( vy cfv cbs wcel eqid cplusf eqtri vx csn cxp cima cv cmulr co wceq wrex
+      crglmod csca crg rlmsca fveq2d syl5eq rexeqdv cmgp mgpbas mgpplusg fveq2i
+      syl ssidd elgrpsumsetsn clmod wb rlmlmod rlmbas rlmvsca crsp clspn rspval
+      lspsnel syl2anc 3bitr4d eqrdv ) AUADBGUBZUCUDZVPFOZAUAUEZNUEGCUFOZUGUHZNB
+      UIWANCUJOZUKOZPOZUIZVSVQQVSVRQZAWANBWDABCPOZWDHACWCPACULQZCWCUHLCULUMVAUN
+      UOUPANBBVTDCUQOZGVSBCWIWIRZHURCVTWIWJVTRUSDESOWISOJEWISIUTTABVBMVCAWBVDQZ
+      GBQWFWEVEAWHWKLCVFVAMVTVSNWCWDFBWBGWCRWDRBWGWBPOHCVGTCVHFCVIOWBVJOKCVKTVL
+      VMVNVO $.
+
+    $( The sumset of a ring with a single element is a principal ideal.
+       (Contributed by Thierry Arnoux, 21-Jan-2024.) $)
+    sumsetsnidl $p |- ( ph -> ( .X. " ( B X. { X } ) ) e. ( LPIdeal ` R ) ) $=
+      ( vx csn cxp cfv wcel wceq wb clpidl fveq2d eqeq2d sumsetsnpridl rspcedvd
+      cima cv wrex sneq adantl crg eqid islpidl syl mpbird ) ADBGOZPUFZCUAQZRZU
+      QNUGZOZFQZSZNBUHZAVCUQUPFQZSZNGBMUTGSZVCVFTAVGVBVEUQVGVAUPFUTGUIUBUCUJABC
+      DEFGHIJKLMUDUEACUKRUSVDTLBURCNUQFURULKHUMUNUO $.
+  $}
+
+  ${
+    $d .+^ f x y z $.  $d .0. i $.  $d B f i x y $.  $d I f i x y z $.
+    $d J f i x y z $.  $d K f i x y z $.  $d R f i x y $.  $d f i ph x y z $.
+    sumsetidl.1 $e |- B = ( Base ` R ) $.
+    sumsetidl.3 $e |- .+^ = ( +f ` R ) $.
+    sumsetidl.4 $e |- K = ( RSpan ` R ) $.
+    sumsetidl.5 $e |- ( ph -> R e. Ring ) $.
+    sumsetidl.6 $e |- ( ph -> I e. ( LIdeal ` R ) ) $.
+    sumsetidl.7 $e |- ( ph -> J e. ( LIdeal ` R ) ) $.
+    $( The sumset sum of two ideals is the ideal generated by their union.
```

Comment:
The "sumset sum" is excessive, I think it's better to write "The sum of two ideals ...".

Reply:
Done with last commit.